src/phoenix/core/evals.py

```diff
@@ -171,3 +173,58 @@ def get_document_evaluation_scores(
                 if result.HasField("score") and document_position < num_documents:
                     scores[document_position] = result.score.value
         return scores
+
+    def export_evaluations(self) -> List[Evaluations]:
+        evaluations: List[Evaluations] = []
+        evaluations.extend(self._export_span_evaluations())
+        evaluations.extend(self._export_document_evaluations())
+        return evaluations
+
+    def _export_span_evaluations(self) -> List[SpanEvaluations]:
+        span_evaluations = []
+        for eval_name, span_evaluations_by_id in self._span_evaluations_by_name.items():
+            span_ids = []
+            rows = []
+            for span_id, pb_eval in span_evaluations_by_id.items():
+                result = pb_eval.result
+                row = {}
+                if result.HasField("score"):
+                    row["score"] = result.score.value
+                if result.HasField("label"):
+                    row["label"] = result.label.value
+                if result.HasField("explanation"):
+                    row["explanation"] = result.explanation.value
```

Comment:
does this work?
```suggestion
                row = MessageToDict(pb_eval.result)
```

Reply:
Good suggestion! It worked. Addressed in f0119440.